_partials/_migrate_live_setup_connection_strings.md

```diff
@@ -8,6 +8,4 @@ export TARGET="postgres://tsdbadmin:<PASSWORD>@<HOST>:<PORT>/tsdb?sslmode=requir
 You find the connection information for your Timescale Cloud service in the configuration file you
 downloaded when you created the service.
 
-<Highlight type="important">
-Avoid using connection strings that route through connection poolers like PgBouncer or similar tools. The live-migration tool requires a direct connection to the database to function properly.
-</Highlight>
+Avoid using connection strings that route through connection poolers like PgBouncer or similar tools. This tool requires a direct connection to the database to function properly.
```

Comment:
This is an important point which I feel must be highlighted. Otherwise users tend to ignore and think using connection pooler is always better and end up in trouble.

Reply:
Done